objects/client.c

```diff
@@ -387,11 +387,11 @@
  *
  * **Signal:**
  *
- *  * *property::icon*
+ *  * *property::icon\_sizes*
```

Comment:
Thanks!

What about it? `c.icon_name` is unrelated to `c.icon`.

The "icon" in "icon_name" refers to "iconified" which basically means "minimized". So `c.name` is the normal name and `c.icon_name` is the name while iconified.